airflow/providers/google/cloud/triggers/bigquery.py

```diff
@@ -49,6 +59,9 @@ class BigQueryInsertJobTrigger(BaseTrigger):
 
     def __init__(
         self,
+        dag_id: str,
+        task_id: str,
+        run_id: str | None,
```

Comment:
This might cause existing triggers to fail since there is change in signature and restart will have old serialised method value in database without these arguments.

Reply:
Removed it completely now. I am using task_instance from BaseTrigger directly: https://github.com/apache/airflow/pull/39442/commits/61160d8196f809738414f1ec8cf92c83db780e06